README.md

```diff
@@ -0,0 +1,40 @@
+# go bindings for Elektra
+
+This repository contains the low level ("kdb" subpackage) and high level (root package, TODO) go bindings for the elektra library.
+
+## Execute tests
+Prerequisite: you have to have KDB and go installed on your machine.
+
+Execute all tests:
+`go test ./...`
+
+Execute tests of a package, e.g. kdb:
+`go test ./kdb`
+
+## Use Elektra in your application
+Just *go get* it like you are used to with go.
+
+`go get github.com/ElektraInitiative/go-elektra`
```

Comment:
When tricking go to use the "bindings" branch, I get the error:

```
../../../../go/src/github.com/ElektraInitiative/go-elektra/elektra.go:45:133: cannot use errCallback (type func(*_Ctype_struct__ElektraError)) as type *[0]byte in argument to func literal
```

go version go1.11.6 linux/amd64

Reply:
yes thats because I pushed a commit that broke the build, should work again

Reply:
Maybe we should rethink our branches strategy: the master should contain what I should review and you make PRs to a pr-branch (which is just there so that you can make a PR). And the actual works happens in feature branches as the go develop model suggests (to be merged into master).

What do you think?

Reply:
sounds good!

Reply:
Yes, these 3 steps sound fine.

> Are you not contradicting yourself?

These two statements have nothing to do with each other. The first statement is about the review, the second is about how you can develop. I am also ok with you directly pushing to master, as I will only review on request.